setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -55,15 +55,16 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $errors = $this->validate($input);
+        $data = $this->getUserData($input, $output);
+        $errors = $this->validate($data);
         if ($errors) {
             $output->writeln('<error>' . implode('</error>' . PHP_EOL .  '<error>', $errors) . '</error>');
             return;
         }
         $installer = $this->installerFactory->create(new ConsoleLogger($output));
-        $installer->installAdminUser($input->getOptions());
+        $installer->installAdminUser($data);
         $output->writeln(
-            '<info>Created Magento administrator user named ' . $input->getOption(AdminAccount::KEY_USER) . '</info>'
+            '<info>Created Magento administrator user named ' . $data[AdminAccount::KEY_USER] . '</info>'
```

Comment:
At this point of execution it is guaranteed that this key exists in the array.  It has gone through the optional prompting step and has been validated with `addUserInfoRules()`.